Resources/doc/reference/installation.rst

```diff
@@ -55,10 +56,15 @@ You will need to configure SonataArticleBundle's entities, if you plan to use th
                 article:  Application\Sonata\ArticleBundle\Entity\Article
                 fragment: Application\Sonata\ArticleBundle\Entity\Fragment
 
-            fragments: # this is currently in development, hope we'll get this soon :)
+            fragments:
                 sonata.article.fragment.title:
                 sonata.article.fragment.text:
 
+.. note::
+
+    We plan to improve the fragments available to allow a configuration for each article type.
+    For example, you will want the fragment 'Comments' only on articles of type 'Blog'.
+
```

Comment:
To be honest, I haven't thought a lot about it, it's something we want to add, but seems hard because we don't have a service per Article Type.
Actually the type is only a property of article. I don't know the best way to do this.